docs/markdown/snippets/fstrings.md

```diff
@@ -0,0 +1,7 @@
+## Introducing format strings to the Meson language
+
+In addition to the conventional `'A string @0@ to be formatted @1@'.format(n, m)`
+method of formatting strings in the Meson DSL, there's now the additional
```

Comment:
likewise here, when you're not into programming languages, "DSL" could be a bit cryptic?